packages/gateway/src/location.ts

```diff
@@ -28,6 +28,24 @@ export const fetchLocation = async (id: UUID) => {
   return response.json() as Promise<Location>
 }
 
+const FETCH_ALL_LOCATIONS = new URL(
+  '/locations?type=ADMIN_STRUCTURE',
```

Comment:
Does Hearth enforce a limit of results by default? I remember I might have had that issue once

Reply:
True but I'm not querying to hearth so I'm hoping application config already takes care of that :smile:

Reply:
Okay apparently it doesn't :man_facepalming: . I'll update it, thanks Riku!